packages/node/example/multiSession/src/serverSideApp.ts

```diff
@@ -0,0 +1,128 @@
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// We only need the following imports from the Node AuthN library.
+import {
+  getSessionFromStorage,
+  Session,
+  getSessionIdFromStorageAll,
+} from "../../../dist/Session";
+
+import cookieSession from "cookie-session";
+
+const clientApplicationName = "solid-client-authn-node multi session demo";
+
+import express from "express";
+const app = express();
+const PORT = 3001;
+
+const DEFAULT_OIDC_ISSUER = "https://broker.pod.inrupt.com/";
+
+app.use(
+  cookieSession({
+    name: "session",
+    // These keys are required by cookie-session to sign the cookies.
+    keys: ["key1", "key2"],
+    maxAge: 24 * 60 * 60 * 1000, // 24 hours
+  })
+);
+
+app.get("/", async (req, res, next) => {
+  const sessionIds = await getSessionIdFromStorageAll();
+  const sessions = await Promise.all(
+    sessionIds.map(async (sessionId) => {
+      return await getSessionFromStorage(sessionId);
+    })
+  );
+  const htmlSessions =
+    sessions.reduce((sessionList, session) => {
+      if (session?.info.isLoggedIn) {
+        return sessionList + `<li>${session?.info.webId}</li>`;
+      }
+      return sessionList + "<li>Anonymous</li>";
+    }, "<ul>") + "</ul>";
+  res.send(
+    `<p>There are currently ${sessionIds.length} visitors: ${htmlSessions}</p>`
+  );
+});
+
+app.get("/login", async (req, res, next) => {
+  const session = new Session();
+  req.session!.sessionId = session.info.sessionId;
+  await session.login({
+    redirectUrl: "http://localhost:3001/redirect",
+    oidcIssuer: DEFAULT_OIDC_ISSUER,
+    clientName: clientApplicationName,
+    handleRedirect: (redirectUrl) => res.redirect(redirectUrl),
+  });
+  if (session.info.isLoggedIn) {
+    res.send(`<p>Already logged in with WebID: [${session.info.webId}]</p>`);
+  }
+});
+
+app.get("/redirect", async (req, res) => {
+  const session = await getSessionFromStorage(req.session!.sessionId);
+  if (session === undefined) {
+    res
+      .status(400)
+      .send(`<p>No session stored for ID ${req.session!.sessionId}</p>`);
+  } else {
+    await session.handleIncomingRedirect(getRequestFullUrl(req));
+    if (session.info.isLoggedIn) {
+      res.send(`<p>Logged in as [${session.info.webId}] after redirect</p>`);
+    } else {
+      res.status(400).send(`<p>Not logged in after redirect</p>`);
+    }
+  }
+  res.end();
+});
+
+app.get("/fetch", async (req, res, next) => {
+  const session = await getSessionFromStorage(req.session!.sessionId);
+  if (!req.query["resource"]) {
+    res.status(400).send("<p>Expected a 'resource' query param</p>");
```

Comment:
Would be nice to explain this a little, and offer an example (as anyone trying this would, I suspect, first try with no query param at all):
```suggestion
    res.status(400).send("<p>Expected a 'resource' query param, for example <strong>http://localhost:3001/fetch?resource=https://pod.inrupt.com/MY_USERNAME/</strong> to fetch the resource at the root of your Pod (which, by default, only <strong>you</strong> will have access to).</p>");
```